addon/components/frost-list.js

```diff
@@ -125,6 +180,43 @@ export default Component.extend({
     })
   },
 
+  setDefaultItem () {
+    const item = this.get('item')
+    const componentKeyNamesForTypes = this.get('componentKeyNamesForTypes')
+
+    if (item && !componentKeyNamesForTypes) {
+      const componentKeyNames = this.get('componentKeyNames')
+      const itemComponentKey = get(componentKeyNames, 'item')
+      let componentName = getComponentName(item)
```

Comment:
@jscharf What is the value being used for?  What is the purpose/context in which this method is used?